package.json

```diff
@@ -15,12 +15,13 @@
     "test:proposals": "forge test --match-path './test/proposals/**' --fork-url $ETH_RPC_URL -vvv",
     "test:integration": "forge test --match-path './test/integration/**' --fork-url $ETH_RPC_URL -vvv",
     "coverage": "forge coverage --fork-url $ETH_RPC_URL --match-path './test/**' --report lcov --report summary",
-    "coverage:unit": "forge coverage --match-path './test/unit/**' --report lcov --report summary",
+    "coverage:unit": "forge coverage --match-path './test/unit/**' --report summary --report lcov",
     "coverage:integration": "forge coverage --match-path './test/integration/**' --fork-url $ETH_RPC_URL --report lcov --report summary",
     "fork": "anvil --fork-url $ETH_RPC_URL --block-time 10 --chain-id 1337",
     "deploy:local": "ETH_PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80 forge script scripts/DeployProposal.s.sol:DeployProposal -vvvv --rpc-url http://127.0.0.1:8545 --broadcast",
     "prettier": "npx prettier --check 'src/**/*.sol'",
-    "prettier:fix": "npx prettier --write 'src/**/*.sol'"
+    "prettier:fix": "npx prettier --write 'src/**/*.sol'",
+    "prettier:watch": "onchange \"(src|test)/**/*.sol\" -- prettier --write {{changed}}"
```

Comment:
added prettier on the tests for this npm script but not the other, maybe we should apply prettier to the test directory. I didn't want to commit a big diff